po/es.po

```diff
@@ -4068,39 +4077,43 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
-"```rust,editable\n"
-"fn main() {\n"
-"    println!(\"Modifícame!\");\n"
-"}\n"
-"```"
 
 #: src/control-flow/blocks.md:43 src/enums.md:34 src/enums/sizes.md:28
 #: src/pattern-matching.md:25 src/pattern-matching/match-guards.md:22
 #: src/structs.md:31 src/methods.md:30 src/methods/example.md:46
 msgid "Key Points:"
-msgstr ""
+msgstr "Puntos Clave:"
 
 #: src/control-flow/blocks.md:44
 msgid ""
 "The point of this slide is to show that blocks have a type and value in "
 "Rust. "
 msgstr ""
+"El objetivo de esta diapositiva es mostrar que los bloques tienen un tipo y "
+"un valor en Rust. "
 
 #: src/control-flow/blocks.md:45
 msgid ""
 "You can show how the value of the block changes by changing the last line in "
 "the block. For instance, adding/removing a semicolon or using a `return`."
 msgstr ""
+"* El objetivo de esta diapositiva es mostrar que los bloques tienen un tipo "
```

Comment:
not sure why do we have here a bulleted list. Also the second bullet lacks a newline; and the original did not had a bulleted list.

Reply:
That's a mistake after the normalization.